src/vanilla/internals.ts

```diff
@@ -320,17 +312,22 @@ function getMountedOrPendingDependents(
 type StoreHook = {
   (): void
   add(callback: () => void): () => void
+  callbacks: SetLike<() => void>
```

Comment:
To me, this feels kind of overwrapping.
If we aren't be afraid of breaking changes (which is the purpose of "internals" and "Rev"), I hope this to be:
```suggestion
  hooks: SetLike<() => void>
```